### ISSUE TITLE
Setup E2E pipeline

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#29350]
### User Story

As a Dev/QA-engineer, I want to be able to have the E2E tests that I write and merge into trunk executed in the pipeline similarly to the other test types. 
This needs to include displaying the results in a user-friendly format and enough logging to troubleshoot in case of failures. 

### Acceptance Criteria

E2E pipeline in place including:

- [ ] Execute all our E2E tests merged into trunk
- [ ] Displaying results in a user-friendly format
- [ ] Enough logging to be able to troubleshoot upon failures
- [ ] Optimized so they can run as fast as possible (run in parallel?) 

### Proposed Objective

Core Features

### Proposed Priority

Priority 2 - Important

```[tasklist]
### Tasks
- [ ] https://github.com/dotCMS/core/issues/29845
- [ ] https://github.com/dotCMS/core/issues/29846
- [ ] https://github.com/dotCMS/core/issues/29847
- [ ] https://github.com/dotCMS/core/issues/29848
```

**Comment by dsilvam** [comment]
IQA done on each of the subtasks

**Comment by josemejias11** [comment]
**Approved**:
